code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -185,9 +185,9 @@ GLOBAL_LIST_INIT(strippable_monkey_items, create_strippable_list(list(
 /mob/living/carbon/monkey/angry
 	ai_controller = /datum/ai_controller/monkey/angry
 
-/mob/living/carbon/monkey/angry/Initialize(mapload)
+/mob/living/carbon/monkey/angry/Initialize(mapload, roll_hat = TRUE)
```

Comment:
Use a variable on the monkey rather than an proc variable, then make a subtype of monkey that is hatless.